server/src/main/java/org/opensearch/action/admin/cluster/remote_store/restore/RestoreRemoteStoreAction.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.admin.cluster.remote_store.restore;
```

Comment:
Will `remotestore` be a better naming convention for the package compared to `remote_store` ?

Reply:
Agree, will get it changed.